exporter/tencentcloudlogserviceexporter/uploader.go

```diff
@@ -45,8 +44,8 @@ type logServiceClientImpl struct {
 
 // newLogServiceClient Create Log Service client
 func newLogServiceClient(config *Config, logger *zap.Logger) (logServiceClient, error) {
-	if config == nil || config.TCPAddr.Endpoint == "" || config.LogSet == "" || config.Topic == "" {
-		return nil, errors.New("missing logservice params: TCPAddr, LogSet, Topic")
+	if config == nil || config.Region == "" || config.LogSet == "" || config.Topic == "" {
```

Comment:
Could you please create an issue to move this logic to the `Validate` function?

Reply:
Sure.